tests/ballerina-integration-test/src/test/java/org/ballerinalang/test/service/http/sample/HTTPHeaderServerTestCase.java

```diff
@@ -0,0 +1,126 @@
+/*
```

Comment:
Now we have a module called jBallerina-integration-test to add integration tests. Can you move this test to that module?